tests/components/bangolufsen/util.py

```diff
@@ -0,0 +1,17 @@
+"""Tools for testing bangolufsen."""
+
+from homeassistant.components.bangolufsen.const import DOMAIN
+
+from .const import TestConstantsConfigFlow as tc
+
+from tests.common import MockConfigEntry
+
+
+def mock_entry() -> MockConfigEntry:
```

Comment:
Ah you do have some kind of fixture. Can you transform this into a fixture? Also move this to `conftest.py`

Reply:
Done